department-of-veterans-affairs/caseflow
# Automatically add BGS VSO team members to the VSO team in Caseflow

Resolves #11041

### Description
Automatically adds BGS VSO team members to the VSO team in Caseflow when they visit the VSO team page.

### Testing Plan
1. Create a new VSO user 
```ruby
> u = User.create!(css_id: "NEW_VSO", station_id: 101, roles: ["VSO"])
```
2. Login as user
3. Check to make sure the user is not a part of the VSO team
```ruby
> u.organizations
=> []
```
4. Visit the VSO team page at http://localhost:3000/organizations/veterans-service-organization
5. Check to make sure the user is now a part of the VSO team
```ruby
> u.reload.organizations
=> [#<Vso:0x00007fc2187ed948
  id: 6,
  name: "VSO",
  participant_id: "2452415",
  role: "VSO",
  type: "Vso",
  url: "veterans-service-organization">]
```
